lacework/resource_lacework_data_export_rule.go

```diff
@@ -0,0 +1,210 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkDataExportRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkDataExportRuleCreate,
+		Read:   resourceLaceworkDataExportRuleRead,
+		Update: resourceLaceworkDataExportRuleUpdate,
+		Delete: resourceLaceworkDataExportRuleDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkDataExportRule,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "The name of the data export rule.",
+				Required:    true,
+			},
+			"integration_ids": {
+				Type:        schema.TypeList,
+				Required:    true,
+				Description: "The list alert channel ids for the rule to use.",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					StateFunc: func(val interface{}) string {
+						return strings.TrimSpace(val.(string))
+					},
+				},
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Description: "The summary of the data export rule.",
+				Optional:    true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"profile_versions_computed": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					StateFunc: func(val interface{}) string {
+						return strings.TrimSpace(val.(string))
+					},
+				},
+			},
+			"guid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_or_updated_time": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_or_updated_by": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceLaceworkDataExportRuleCreate(d *schema.ResourceData, meta interface{}) error {
+	var (
+		lacework   = meta.(*api.Client)
+		exportRule = api.DataExportRule{
+			Filter: api.DataExportRuleFilter{
+				Name:            d.Get("name").(string),
+				Description:     d.Get("description").(string),
+				ProfileVersions: []string{"V1"},
```

Comment:
silly question but, do we need this?

Reply:
Just looked at this, currently profileVersion is part of RuleDTO.filters but we are not using it anywhere (other than surfacing that in UI).

<img width="388" alt="image" src="https://user-images.githubusercontent.com/99691476/186713859-aaf3f154-6e6a-45bb-9a63-6e3b653c7451.png">
<img width="411" alt="image" src="https://user-images.githubusercontent.com/99691476/186713906-cffebea8-00f9-4933-b27d-423653ce8a47.png">

Reply:
A new data export rule can be created without passing this field:
<img width="1486" alt="image" src="https://user-images.githubusercontent.com/99691476/186716098-4d706d2d-fad4-42ae-a98c-e45639d2e005.png">